web/libs/editor/src/components/Comments/Comment/CommentForm.tsx

```diff
@@ -114,36 +127,69 @@ export const CommentForm: FC<CommentFormProps> = observer(({ commentStore, annot
 
   const currentLinkingComment = annotationStore.selected.currentLinkingMode?.comment;
   const currentComment = getCurrentComment();
-  const { text = "", regionRef } = currentComment || {};
+  const { text = "", regionRef, classifications } = currentComment || {};
   const { region, result } = regionRef || {};
   const linking = !!linkingComment && currentLinkingComment === linkingComment && globalLinking;
   const hasLinkState = linking || region;
+  const selections = taxonomyPathsToSelectedItems(classifications?.default?.values);
+  const classificationsItems = parseCommentClassificationConfig(commentStore.commentClassificationConfig);
```

Comment:
we should not make calculations like this directly in render. At least `useMemo()` should be used, but in this case it should be moved to a getter in comment store, because it will be calculated only once.